Workade/Views&ViewModels/Map/MapViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  MapViewController.swift
+//  Workade
+//
+//  Created by Inho Choi on 2022/10/23.
+//
+
+import NMapsMap
+import UIKit
+
+class MapViewController: UIViewController {
+    lazy var map = NMFMapView()
+    private var topInfoStackView: UIStackView = {
+        var stackView = UIStackView()
+        stackView.backgroundColor = .clear
+        stackView.axis = .horizontal
+        stackView.alignment = .trailing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return stackView
+    }()
+    private var officeNameLabel: BasePaddingLabel = {
+        var label = BasePaddingLabel(padding: UIEdgeInsets(top: 12, left: 40, bottom: 12, right: 40))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "제주 O-PEACE"
+        label.textAlignment = .center
+        label.backgroundColor = .theme.background
+        label.layer.masksToBounds = true
+        label.layer.cornerRadius = 20
+        label.font = UIFont.customFont(for: .subHeadline)
+        
+        return label
+    }()
+    private lazy var backButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .white
+        button.setTitle("X", for: .normal)
```

Comment:
```suggestion
        let imageConfig = UIImage.SymbolConfiguration(pointSize: 17, weight: .bold)
        let xmarkImage = UIImage(systemName: "xmark", withConfiguration: imageConfig)
        button.setImage(xmarkImage, for: .normal)
```
[P2] 참고사항의 xmark를 이렇게 구현해주시면 감사하겠습니다!!

Reply:
iPhone 14 Pro, iPhone 13(실기기), iPhone 8에서 레이아웃 깨짐 없음. 정상작동을 확인했습니다!

Reply:
이 부분은 제가 못봤군요 수정하겠습니다.